packages/cactus-plugin-ubiquity/src/main/typescript/generated/openapi/typescript-axios/.npmignore

```diff
@@ -0,0 +1 @@
+# empty npmignore to ensure all required files (e.g., in the dist folder) are published by npm
```

Comment:
@RafaelAPB Please delete this file - it is ignored in the open api generator ignore file as well.